src/app/components/toolbar/toolbar.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { ReadfFileService } from '../../services/readfFile.service';
+import { Folder } from '../../interfaces/Folder';
+
+@Component({
+  selector: 'app-toolbar',
+  templateUrl: './toolbar.component.html',
+  styleUrls: ['./toolbar.component.scss']
+})
+export class ToolbarComponent {
+  arr!: Folder[]
+
+  constructor(private readFileService: ReadfFileService) {
+    this.arr = readFileService.readFile();
```

Comment:
let's make it more readable: `this.filesStructure = filesStructureService.getFilesStructure();`